demos/storybook/package.json

```diff
@@ -12,7 +12,7 @@
         "@pxblue/react-themes": "^6.0.0",
         "@pxblue/storybook-rtl-addon": "^1.0.0",
         "@pxblue/storybook-themes": "^1.1.0",
-        "@sambego/storybook-state": "^1.3.6",
+        "@sambego/storybook-state": "^2.0.0",
```

Comment:
If we're refactoring to local state, I don't think we need this package at all anymore

Reply:
We're using this for the with-custom-menu UserMenu story, so I'm going to leave this in here for now unless we refactor that to use local state later.